### DISCUSSION TITLE
Querying a non public schema using python client

### DISCUSSION THREAD

**Discussion by footballdotpy** [supabase/supabase#26894] [Questions]
Evening all,

I have read up that this is a known issue with postgres and was hoping there would be a workaround for it.

I have data in a schema that is not the public schema and wish to query the data in python.

the exception thrown is this 
postgrest.exceptions.APIError: {'code': '42P01', 'details': None, 'hint': None, 'message': 'relation "public.trading.country_names" does not exist'}

the default query points to public and I need to reroute this to the trading schema

any solution available?

**Comment by GaryAustin1** [comment]
2 day old change...
https://github.com/supabase-community/supabase-py/issues/808
And older way?
https://github.com/supabase-community/supabase-py/issues/618
